pkg/api/grpcServer.go

```diff
@@ -76,6 +83,10 @@ type GRPCServerOptions struct {
 	// AttributeManager holds a pointer to an initialized AttributeManager to use when
 	// processing incoming attribute requests.
 	AttributeManager attribute.Manager
+
+	// Tracer is the tracer instance to use with OpenTracing. If EnableTracing is false
```

Comment:
Instead of EnableTracing and this field, can we just have this field and compare it to nil to see if tracing is desired or not?